libraries/aws_iam_root_user.rb

```diff
@@ -20,6 +20,16 @@ def has_mfa_enabled?
     summary_account['AccountMFAEnabled'] == 1
   end
 
+  def has_hardware_mfa_enabled?
```

Comment:
I recommend flipping this method, as mentioned on slack: `has_virtual_mfa_enabled?` should return `true` if any devices match the pattern.